### ISSUE TITLE
IPsec configuration not working with CARP IP

### ISSUE THREAD

**Issue by fraenki** [opnsense/core#267]
OPNsense 15.7.4

The IPsec configuration will NOT be written to `/usr/local/etc/ipsec.conf` if you use a CARP address as interface for the Phase 1 configuration.

When using the _Connect VPN_ button in the GUI, the log will only have these entries:
```
Jul 21 20:47:19 	charon: 03[CFG] no config named 'con1'
Jul 21 20:47:19 	charon: 03[CFG] received stroke: initiate 'con1'
Jul 21 20:47:19 	charon: 16[CFG] no IKE_SA named 'con1' found
Jul 21 20:47:19 	charon: 16[CFG] received stroke: terminate 'con1'
```
Nothing else will be logged and no connection attempt will be made. The file `/usr/local/etc/ipsec.conf` will not contain the VPN configuration.

If you change the configuration to use the WAN interface, these log entries appear (and the VPN :
```
Jul 21 20:49:08 	charon: 09[CFG] received stroke: route 'con1'
Jul 21 20:49:08 	charon: 03[CFG] added configuration 'con1'
Jul 21 20:49:08 	charon: 03[CFG] received stroke: add connection 'con1'
```
Attempts to connect to the VPN will start immediately.

**Comment by AdSchellevis** [comment]
Thanks for reporting!
commit https://github.com/opnsense/core/commit/b84506acd042a348fa0a776d15b49f4bfe6d6809 should fix the missing CARP IP

**Comment by fraenki** [comment]
Thanks for the quick fix @AdSchellevis! Any chance that this fix will make it into a 15.7.x release?

**Comment by AdSchellevis** [comment]
Hi @fraenki it will be in the next release, probably this week. ( @fichtner already git cherry picked it in)
